src/views/welcome-view/WelcomeView.vue

```diff
@@ -236,7 +239,17 @@ export default defineComponent({
         .dispatch("logIn", {
           password: this.password,
         })
-        .then(() => this.$router.push("/"))
+        .then(async () => {
```

Comment:
Don't need async here as there is no promise being executed here.

Reply:
@jdeepee Will the UNLOCK_AGENT mutation throw an error if the password is wrong?

Reply:
@leifriksheim no it will just return isUnlocked = false in the response